README.md

```diff
@@ -244,6 +244,10 @@ Kubernetes users:\
 
 Most plugins should be supported out-of-the-box, or maybe require some minimal changes. See this [dashboard](https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=18341) for known compatibility issues.
 
+## Plugin developers
```

Comment:
Should I change the title to "Other plugins' developers"? Or "Third party plugin developers", or whatever you think it's best.

Reply:
what about?
```suggestion
## Adding JCasC support to a plugin
```